jupyter_dash/jupyter_app.py

```diff
@@ -302,25 +303,32 @@ def run(
 
         err_q = queue.Queue()
 
+        server = make_server(
+            host, port, self.server,
+            threaded=True,
+            processes=0
+        )
+        logging.getLogger("werkzeug").setLevel(logging.ERROR)
+
         @retry(
             stop_max_attempt_number=15,
             wait_exponential_multiplier=100,
             wait_exponential_max=1000
         )
         def run():
             try:
-                super_run_server(**kwargs)
```

Comment:
these `**kwargs` were previously used to enable devtools, right? Can we just explicitly call `self.enable_dev_tools` with the appropriate subset of kwargs? And then perhaps throw an error if there are any unused kwargs left?

Reply:
Added `enable_dev_tools` call. :ok_hand: